### ISSUE TITLE
Метод getOrderForApi

### ISSUE THREAD

**Issue by spiritwebdev** [MirAlexSky/ozon-logistics-api#3]
Отсутсвует метод getOrderForApi?
Верно же я понимаю что заказ отправляется через метод uploadSending?

**Comment by MirAlexSky** [comment]
Да, всё верно. Метод будет добавлен. На данный момент вы можете посмотреть данные о заказе, повторно его выгрузив. Озон в этом случае возвращает данные существующего заказа

**Comment by spiritwebdev** [comment]
Так его невозможно создать получается ведь?
Если через uploadSending создавать заказ нужно, он не создается из-за отсутствия метода getOrderForApi)
Сейчас создавать заказы получается не предоставляется возможным?

**Comment by MirAlexSky** [comment]
А зачем вам метод getOrderForApi? Заказ создать можно, метод uploadSending позволяет создать заказ и возвращает при этом данные о созданном заказе или ошибку. Входные данные - массив с данными. Структуру вы знаете, так ведь? Повторюсь, можно взять шаблон массива из ветки master /src/OzonDTO/OzonOrder.php

**Comment by spiritwebdev** [comment]
В методе uploadSending как раз и используется ведь getOrderForApi. На него у меня и ругалось при создании заказа)

**Comment by MirAlexSky** [comment]
А, вот в чём дело. Ветка master является dev версией, стабильная версия находится на ветке 1.x. В master может что-то временно работать не полностью
